.github/workflows/inspektor-gadget.yml

```
@@ -200,13 +203,19 @@ jobs:
         tags: ${{ secrets.CONTAINER_REPO }}:${{ env.IMAGE_TAG }}
         cache-from: type=local,src=/tmp/.buildx-cache
         cache-to: type=local,dest=/tmp/.buildx-cache-new
+        # For the moment, we only support these two platforms.
+        # Cross building for arm64 only happens on main branch.
+        platforms: ${{ fromJSON('["linux/amd64", "linux/amd64,linux/arm64"]')[github.ref == 'refs/heads/francis/arm'] }}
```

From comment https://github.com/kinvolk/inspektor-gadget/pull/441#discussion_r863132828, it will need to be changed to something like:

```suggestion
        platforms: ${{ fromJSON('["linux/amd64", "linux/amd64,linux/arm64"]')[github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v')  ] }}
```

* Support for both main and releases
* 'refs/heads/main' instead of `main`

Ditto below.